src/components/molecules/ArticleListItem/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Heading, Stack, HStack, VStack } from '@chakra-ui/react'
+import { Article } from '@/libs/microcms'
+import Link from 'next/link'
+import Image from 'next/image'
+import PublishedDate from '../PublishedDate'
+
+type Props = {
+  article: Article
+}
+
+export default function ArticleListItem({ article }: Props) {
+  return (
+    <Box key={article.id}>
```

Comment:
Most of the components here is from chakra ui.
https://chakra-ui.com/docs/components